CHANGELOG.md

```diff
@@ -3,6 +3,22 @@ Changelog for NeoFS LOCODE database
 
 ## [Unreleased]
 
+## [0.7.0] - 2024-08-28
+
+### Changed
+- Go 1.22 is required to build now (#40, #43)
+- Corrected known bad coordinates via a local override (#38)
+- Drop cobra dependency, so there is no CLI (#42)
```

Comment:
> there is no CLI

i would not understand it as an external reader. cli is the same and we still have it but it uses standard library now